client/components/Navbar.vue

```diff
@@ -54,7 +54,7 @@ export default class Navbar extends Vue {
   @MenuGetter menus
 
   async beforeMount () {
-    let {data: menus} = await axios.get('/hpi/menus')
+    let {data: menus} = await axios.get('/hpi/ui/menu')
```

Comment:
I'm not sure if your typical project would all have `/hpi` for ALL endpoints, but since this endpoint seemed specific for being used as a UI data source, I've taken the liberty to make this distinction here.

Reply:
It looks good.